messages/src/main/java/life/genny/messages/managers/QEmailMessageManager.java

```diff
@@ -277,12 +277,17 @@ public void sendMessage(BaseEntity templateBe, Map<String, Object> contextMap) {
 		contentArray.add(contentJson.build());
 		mailJsonObjectBuilder.add("content", contentArray.build());
 
+	}catch(Exception ex){
```

Comment:
It is better to only catch and handle specific exception types rather than generic exceptions.